src/main/java/ttt/player/GuiPlayerFactory.java

```diff
@@ -0,0 +1,44 @@
+package ttt.player;
+
+import ttt.GameType;
+
+import static ttt.GameType.*;
+import static ttt.player.PlayerSymbol.O;
+import static ttt.player.PlayerSymbol.X;
+
+public class GuiPlayerFactory implements PlayerFactory {
```

Comment:
This is similar to PlayerFactory, but instead of HumanPlayer, it uses GuiHumanPlayer. The last method 'getUnbeatablePlayerFor' is common between both factories, so can be extracted out into a subclass to prevent the duplication.